chainer/training/updaters/multiprocess_parallel_updater.py

```diff
@@ -0,0 +1,223 @@
+import multiprocessing
+
+import six
+
+import chainer
+from chainer.dataset import convert
+from chainer import cuda
+from chainer.training.updater import StandardUpdater, _calc_loss
+
+try:
+    from cupy.cuda import nccl
+    _available = True
+except ImportError:
+    _available = False
+
+
+class _Worker(multiprocessing.Process):
```

Comment:
Is it common to start a class name with underscore? I haven't seen this before. I believe that this style applies for only methods/variables in class. c.f., https://docs.python.org/3.4/tutorial/classes.html#tut-private